pkg/debuginfo/processor.go

```diff
@@ -33,7 +33,8 @@ import (
 
 var errNotFound = errors.New("not found")
 
-type DebugInfo struct {
```

Comment:
I think that `DebugInfo` or `DebugInfo$thing` is a more direct name?

Reply:
The name of the shoudl be consider with the package name. The full name is `debuginfo.Processor`, it's more align with the conventions rather than `debuginfo.DebugInfo`. I'm open to other suggestions though.

Reply:
Some ideas:

`debuginfo.Extractor`
`debuginfo.Finder`
`debuginfo.Manager`

Reply:
Manager sounds good. Because it does all of the above.